documentation/src/main/asciidoc/topics/con_remote_cache.adoc

```diff
@@ -0,0 +1,17 @@
+[id='remote_caches-{context}']
+= Cache Configuration with {brandname} Server
+Caches configure the data container on {brandname} Server.
+
+You create caches at run-time by adding definitions based on `org.infinispan`
+templates or {brandname} configuration through the console, the Command Line
+Interface (CLI), the Hot Rod endpoint, or the REST endpoint.
+
+[IMPORTANT]
+====
+When you create caches at run-time, {brandname} Server replicates your cache
+definitions across the cluster.
+
+You should not add cache definitions to `{server_config}/infinispan.xml`
```

Comment:
I would not be so drastic (should not). Something like:

Configurations declared directly in `infinispan.xml` are not automatically synchronized across the cluster. If you wish to use this method, ensure that the configuration is propagated to all of the nodes in your cluster using configuration management tools like Ansible or Chef.